src/main/java/seedu/address/logic/commands/AddAllocCommand.java

```diff
@@ -75,6 +76,7 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         if (studentsToEdit.size() > 1) {
+            showStudentsWithNameConflicts(model, studentsToEdit);
```

Comment:
Maybe this will do

```suggestion
            List<String> matchedIds = matchedStudents.stream()
                                    .map(Student::getId).map(ID::getValue)
                                    .collect(Collectors.toList());
            Predicate<Student> predicate = new IdContainsKeywordsPredicate(matchedIds);
            model.updateFilteredStudentList(predicate);
```

Same for `AddScoreCommand` and `AddGroupCommand`

Reply:
Sorry I didn't comment on this earlier, but I think what Vani suggested here may be a better approach than `SameNameConflictUtil::showStudentsWithNameConflicts`, since we're kind of doing too many things at once within that method.

On top of that, while I was working on #173, I think instead of creating a `Predicate` from the IDs of the matched students, would it be better to have a `NameEqualsPredicate` with the name argument that the user passed, then calling `model.updateFilteredStudentList` with that predicate? I feel like that's more natural

What do you guys think? @nbvannhi @leonghongfai

Reply:
@ganhongyao @nbvannhi I think its more natural too, I only did it via IDs as the `NameContainsKeywordsPredicate` is unable to match the whole name fully. If we are fully agreeable, I can change it accordingly.

Reply:
I agree too 👍🏻